scripts/domManipulation.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable prefer-const */
+const booksLocalStorage = JSON.parse(localStorage.getItem('books') || '[]');
+let books = booksLocalStorage;
+const booksContainer = document.querySelector('.books-container');
+
+function displayBooks() {
+  if (booksContainer.querySelectorAll('div')) {
+    Array.from(booksContainer.querySelectorAll('div')).forEach(
+      (bookContainer) => {
+        booksContainer.removeChild(bookContainer);
+      }
+    );
+  }
+
+  books.forEach((book) => {
+    const bookContainer = document.createElement('div');
```

Comment:
- [ ] [OPTIONAL] I think it could be better to use Javascript ES6 by refactoring your functions into arrow function, Ex: `const displayBooks = () => { ...content }` 😉